source/develop/release-management/features/storage/incremental-backup.html.md

```diff
@@ -199,47 +205,47 @@ Response:
 <disks>
     <disk>
         ...
-        <backup>incremental|null</backup>
+        <backup>incremental|none</backup>
         ...
-    </disks>
+    </disk>
     ...
 </disks>
 ```
 
 #### Starting backup
 
-Start incremental backup since backup id "previous-backup-uuid".
+Start incremental backup since checkpoint id "previous-checkpoint-uuid".
 
 Request:
 ```
 POST /vms/vm-uuid/backups
 
-<backup>
-    <incremental>previous-backup-uuid</incremental>
+<vm_backup>
```

Comment:
Why vm_backup? We are under "/vms/vm-uuid/backups", so I expect the resource to be "backup".

Reply:
It's auto-generated from the restapi entity name. Didn't find a way to easily change it. I'll give it another try..